libmscore/beam.cpp

```diff
@@ -1532,7 +1532,7 @@ void Beam::layout2(std::vector<ChordRest*>crl, SpannerSegmentType, int frag)
             // (all the chordrests of a beam have the same up/down, as it depends on TAB parameters if there are no voices
             // or from the voice the beam belongs to if there are voices; then, it is enough to check only the first chordrest)
             _up = c1->up();
-            // compute vert. pos. of beam, relative to staff (top line = 0)
+            // compute vert. posWithUserOffset. of beam, relative to staff (top line = 0)
```

Comment:
This comment change doesn't seem to be meaningful. There seem to be many such comments changes within this PR, perhaps the refactoring tool just replaced all `pos` occurrences in comments it could found. But those occurrences are not necessarily related to the name of the function being renamed.